packages/model-serving/src/components/deploymentWizard/fields/modelLocationFields/URIConnectionField.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import {
+  FormHelperText,
+  HelperText,
+  HelperTextItem,
+  TextInput,
+  ValidatedOptions,
+} from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import {
+  ConnectionTypeDataField,
+  UriField,
+} from '@odh-dashboard/internal/concepts/connectionTypes/types';
+import DefaultValueTextRenderer from '@odh-dashboard/internal/concepts/connectionTypes/fields/DefaultValueTextRenderer';
+import {
+  trimInputOnBlur,
+  trimInputOnPaste,
+} from '@odh-dashboard/internal/concepts/connectionTypes/utils';
+import { FieldMode } from '@odh-dashboard/internal/concepts/connectionTypes/fields/types.js';
+
+const validateUrl = (url?: string) => {
+  if (!url) {
+    return true;
```

Comment:
I think it's weird to validate a URL that doesn't exist and return true if it doesn't? I think the url should be mandatory and only run validation if it exists